chalice/app.py

```diff
@@ -350,6 +387,8 @@ def to_dict(self):
         # We want the output of `to_dict()` to be
         # JSON serializable, so we need to remove the CaseInsensitive dict.
         copied['headers'] = dict(copied['headers'])
+        if copied['query_params'] is not None:
```

Comment:
Codecov reports this as never testing the negative case. Can we add a test for that?